trek.js

```diff
@@ -0,0 +1,123 @@
+const url = 'https://trektravel.herokuapp.com/trips';
+
+const scrollUp = function scrollUp() {
+  $('html, #wrapper').animate({ scrollTop: 0 }, 500);
+};
+
+const postSuccessCallback = function postSuccessCallback(response) {
+  console.log('POST request was successful');
+  console.log(response);
+
+  let generatedHMTL = `<p>You have successfully reserved a spot as ${response.name} </p>`;
+  $('.success').html(generatedHMTL).show().delay(1000).fadeOut();
+  scrollUp();
+};
+
+const querySuccessCallback = function querySuccessCallback(response) {
+  $('#show-trip').hide();
+  let body = '';
```

Comment:
I like that you broke this callback out into its own named function - this makes your code much easier to follow. Could you have done the same for the callbacks for loading all trips or for loading trip details?